lib/task-data/tasks/install-centos.js

```diff
@@ -14,20 +14,10 @@ module.exports = {
         installScriptUri: '{{api.templates}}/{{options.installScript}}',
         hostname: 'localhost',
         comport: 'ttyS0',
-        domain: 'rackhd',
         completionUri: 'renasar-ansible.pub',
```

Comment:
If the task schema is enabled, the original `options` property in task definition become the place to set the default value for some of option. So we should not expect the original `options` property will give you the full supported option list for a specific task, we should always turn to lookup the schema for all supported options. This is the next enhancement plan for task schema.